incubator/command-config/src/main/java/io/aklivity/zilla/runtime/command/config/internal/openapi/http/proxy/OpenApiHttpProxyConfigGenerator.java

```diff
@@ -0,0 +1,519 @@
+    {
+        return writeConfig(namespace);
+    }
+
+    private OpenApi parseOpenApi(
+        InputStream inputStream)
+    {
+        OpenApi openApi = null;
+        try
+        {
+            Jsonb jsonb = JsonbBuilder.create();
+            openApi = jsonb.fromJson(inputStream, OpenApi.class);
+            jsonb.close();
+        }
+        catch (Exception ex)
+        {
+            ex.printStackTrace();
+            rethrowUnchecked(ex);
+        }
+        return openApi;
+    }
+
+    private int[] resolveAllPorts()
+    {
+        int[] ports = new int[openApi.servers.size()];
+        for (int i = 0; i < openApi.servers.size(); i++)
+        {
+            Server2 server2 = Server2.of(openApi.servers.get(i));
+            URI url = server2.url();
+            ports[i] = url.getPort();
+        }
+        return ports;
+    }
+
+    private int[] resolvePortsForScheme(
+        String scheme)
+    {
+        requireNonNull(scheme);
+        int[] ports = null;
+        URI url = findFirstServerUrlWithScheme(scheme);
+        if (url != null)
+        {
+            ports = new int[] {url.getPort()};
+        }
+        return ports;
+    }
+
+    private URI findFirstServerUrlWithScheme(
+        String scheme)
+    {
+        requireNonNull(scheme);
+        URI result = null;
+        for (Server server : openApi.servers)
+        {
+            Server2 server2 = Server2.of(server);
+            if (scheme.equals(server2.url().getScheme()))
+            {
+                result = server2.url();
+                break;
+            }
+        }
+        return result;
+    }
+
+    private Map<String, String> resolveSecuritySchemes()
+    {
+        requireNonNull(openApi);
+        Map<String, String> result = new HashMap<>();
+        if (openApi.components != null && openApi.components.securitySchemes != null)
+        {
+            for (String securitySchemeName : openApi.components.securitySchemes.keySet())
+            {
+                String guardType = openApi.components.securitySchemes.get(securitySchemeName).bearerFormat;
+                if ("jwt".equals(guardType))
+                {
+                    result.put(securitySchemeName, guardType);
+                }
+            }
+        }
+        return result;
+    }
+
+    private NamespaceConfig createNamespace()
+    {
+        return NamespaceConfig.builder()
+            .name("example")
+            .inject(namespace -> injectGuard(namespace))
+            .inject(namespace -> injectTcpServer(namespace, "http_server0", "tls_server0"))
+            .inject(namespace -> injectTlsServer(namespace))
+            .binding()
+                .name("http_server0")
+                .type("http")
+                .kind(SERVER)
+                .options(HttpOptionsConfig::builder)
+                    .access()
+                        .policy(CROSS_ORIGIN)
+                        .build()
+                    .inject(options -> injectHttpServerOptions(options))
```

Comment:
Use `.inject(this::injectHttpServerOptions)` instead.

Reply:
done